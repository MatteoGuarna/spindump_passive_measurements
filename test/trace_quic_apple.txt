A test run with draft-20 version of Apple client and server, using IPv6 on a local interface
