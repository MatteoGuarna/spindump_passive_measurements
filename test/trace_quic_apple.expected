[
{ "Event": "new", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-6c6e01a3bfb6eef5 (56348:4433)", "Ts": 1558447548344685, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "change", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548451130, "State": "Starting", "Packets1": 1, "Packets2": 0, "Bytes1": 1264, "Bytes2": 0, "Bandwidth1": 1264, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548451130, "State": "Up", "Right_rtt": 106445, "Packets1": 1, "Packets2": 0, "Bytes1": 1264, "Bytes2": 0, "Bandwidth1": 1264, "Bandwidth2": 0 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548459015, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 7, "Packets2": 7, "Bytes1": 3036, "Bytes2": 4356, "Bandwidth1": 3036, "Bandwidth2": 4356 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548459716, "State": "Up", "Value": 0, "Who": "initiator", "Packets1": 8, "Packets2": 8, "Bytes1": 3131, "Bytes2": 4655, "Bandwidth1": 3131, "Bandwidth2": 4655 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548459846, "State": "Up", "Value": 0, "Who": "initiator", "Packets1": 9, "Packets2": 8, "Bytes1": 3224, "Bytes2": 4655, "Bandwidth1": 3224, "Bandwidth2": 4655 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548460203, "State": "Up", "Value": 0, "Who": "initiator", "Packets1": 10, "Packets2": 8, "Bytes1": 3514, "Bytes2": 4655, "Bandwidth1": 3514, "Bandwidth2": 4655 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548460257, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 11, "Packets2": 8, "Bytes1": 3629, "Bytes2": 4655, "Bandwidth1": 3629, "Bandwidth2": 4655 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548531020, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 11, "Packets2": 9, "Bytes1": 3629, "Bytes2": 4735, "Bandwidth1": 3629, "Bandwidth2": 4735 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548532473, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 11, "Packets2": 10, "Bytes1": 3629, "Bytes2": 4815, "Bandwidth1": 3629, "Bandwidth2": 4815 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548563235, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 11, "Packets2": 11, "Bytes1": 3629, "Bytes2": 4993, "Bandwidth1": 3629, "Bandwidth2": 4993 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548605558, "State": "Up", "Value": 0, "Who": "initiator", "Packets1": 11, "Packets2": 12, "Bytes1": 3629, "Bytes2": 5171, "Bandwidth1": 3629, "Bandwidth2": 5171 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548605709, "State": "Up", "Value": 0, "Who": "initiator", "Packets1": 12, "Packets2": 12, "Bytes1": 3709, "Bytes2": 5171, "Bandwidth1": 3709, "Bandwidth2": 5171 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548676476, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 13, "Packets2": 12, "Bytes1": 3796, "Bytes2": 5171, "Bandwidth1": 3796, "Bandwidth2": 5171 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548676934, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 13, "Packets2": 13, "Bytes1": 3796, "Bytes2": 5251, "Bandwidth1": 3796, "Bandwidth2": 5251 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["::1","::1"], "Session": "142e532ac4566420-55876d62d4dc1fdb (56348:4433)", "Ts": 1558447548677085, "State": "Up", "Value": 0, "Who": "initiator", "Packets1": 13, "Packets2": 14, "Bytes1": 3796, "Bytes2": 5360, "Bandwidth1": 3796, "Bandwidth2": 5360 }
]
